WTWSMS/history/characters/roman.txt

```diff
@@ -5376,6 +5377,7 @@
 }
 1300550 = {
    	name="Tatianus" # consul of 466, PLRE vol. 2 Tatianus 1
+    dynasty=1042325
```

Comment:
What's the point of giving him and 1300551 this dynasty?

Reply:
This is dynasty of their mother and grandfather.

Reply:
Dynasties are not matrilineal, so that does not work.

Reply:
I think it is better than if they get random dynasties.

Reply:
No, random dynasties is more accurate actually.

Reply:
I have opposite opinion on this issue. I think better is add dynasties to all characters as possible.

Reply:
Not if it leads to the setup being less accurate though. Plus that it isn't a big issue to have random in cases where we simply do not know, since a dynasty is attached automatically.

Reply:
In game Tatianus will get one random dynasty, Iulius will get another. If they gets one random dynasty it would be nice. But game mechanic not allow that.

Reply:
I agree with @loup99 here. They shouldn't get the dynasty of their mother. They can be left without a dynasty or they can get a new created one with their father.